### PR TITLE
Dataset for 3D human mesh estimation task.

### PR DESCRIPTION
**Pull request by zengwang430521**
Add the codes for Human3.6M and lsp dataset used in 3D human mesh estimation task.
Add test code and test data for mesh dataset and mesh pipelines.

### CODE REVIEW

File: mmpose/core/evaluation/mesh_eval.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+
+def compute_similarity_transform(S1, S2):
```
Comment (innerlee) [inline mmpose/core/evaluation/mesh_eval.py:4]: is `S` short for Surface?

Comment (zengwang430521) [reply mmpose/core/evaluation/mesh_eval.py:4]: I adapt the codes from SPIN directly.  In SPIN, 'S' is used to represent 3D points and I don't know what it is short for. 
If it's not clear enough, I can change the name later.

Comment (innerlee) [reply mmpose/core/evaluation/mesh_eval.py:4]: need to add licence header if the file is adapted from other repo. see examples [here](https://github.com/open-mmlab/mmpose/blob/7672061b28b9b67482852f8f6f3ac8425026a456/mmpose/core/post_processing/nms.py)

yeah better to change the name to something meaningful

**Comment by innerlee** [comment]
kindly ping, any progress?

**Comment by zengwang430521** [comment]
Hi, I have rewrited the codes for 3D human mesh estimation task and I want to push the codes to the human_mesh branch. 
Could you please delete this branch? Thanks!

**Comment by innerlee** [comment]
simply force push is enough

**Comment by innerlee** [comment]
oh got what you meant

**Pull request closed without merging**